.golangci-strict.yml

```diff
@@ -155,7 +155,7 @@ linters-settings:
   nolintlint:
     # Exclude following linters from requiring an explanation.
     # Default: []
-    allow-no-explanation: [ funlen, gocognit, lll ]
+    allow-no-explanation: [ funlen, gocognit, lll, forbidigo ]
```

Comment:
I think it's not a very good move. By adding this exception we allow debug prints to stay in the code. I think, it's better to replace all `fmt.Print*` with `fmt.Fprintf(os.Stderr...` because it's the correct way to print user messages in CLI utilities. Or use `log`.